### PR TITLE
Implement creating dell overlay with perccli

### PR DESCRIPTION
**Pull request by pengz1**
This PR is to create overlay for dell perccli tool.  perccli is a Dell version MegaRaid/StorCLI tool that will be used to catalog RAID and disks info, create RAID and delete RAID on dell servers.
perccli tool is not included as we still waiting for license approving.

### CODE REVIEW

**Comment by pengz1** [comment]
@iceiilin @RackHD/corecommitters

File: roles/oem/overlay/provision_dell_raid_overlay/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  with_items: raid_overlay_package_manifest
+
+# Install perccli tools
+- name: Copy perccli tool
+  copy: src="{{ playbook_dir }}/files/{{ raid_overlay_perccli_package }}" dest=/opt/
+
+- name: Install perccli tools
+  command: dpkg -i /opt/{{ raid_overlay_perccli_package }}
+
+- name: Remove perccli tool
```
Comment (benbp) [inline roles/oem/overlay/provision_dell_raid_overlay/tasks/main.yml]: Maybe call this "Remove perccli package" so it's not confusing that we're removing the installed tool?

**Comment by benbp** [comment]
:+1:

File: roles/oem/overlay/provision_dell_raid_overlay/tasks/main.yml (hunk 1 above)
Comment (pengz1) [inline roles/oem/overlay/provision_dell_raid_overlay/tasks/main.yml]: Good catch, updated as suggested.

**Comment by iceiilin** [comment]
+1

**Comment by yyscamper** [comment]
:+1:

**Pull request merged**